src/test/java/uk/gov/hmcts/reform/civil/service/DeadlinesCalculatorTest.java

```diff
@@ -420,4 +423,44 @@ void testPlusWorkingDaysReturnsSameDay() {
         LocalDate wednesday = LocalDate.of(2022, 9, 28);
         assertThat(calculator.plusWorkingDays(wednesday, 0)).isWednesday();
     }
+
+    @Nested
+    class GetSLAStartDate {
```

Comment:
no test for if allocatedTrack is empty?

Reply:
Allocated Track should always be set based on earlier steps in the journey

Reply:
https://github.com/hmcts/civil-service/blob/master/src/main/java/uk/gov/hmcts/reform/civil/enums/AllocatedTrack.java#L24

Reply:
It's all good seems i cant test the default step regardless but i have to provide one. In the event of a null allocated track it would break in the switch. Added null checks on the fields that i'm trying to run functionality on. Extra checks can't hurt :)